.gitignore

```diff
@@ -4,6 +4,7 @@
 *.class
 bin/main/application.yaml
 **/.terraform
+.env.local
```

Comment:
This should be left it - it's meant to act as a template to copy and paste to .env and fill in to work locally. It shouldn't have any values in it, just prompts for what's needed